pkg/virt-api/rest/definitions.go

```diff
@@ -293,6 +293,14 @@ func NamespaceParam(ws *restful.WebService) *restful.Parameter {
 	return ws.PathParameter("namespace", "Object name and auth scope, such as for teams and projects").Required(true)
 }
 
+func ForceParam(ws *restful.WebService) *restful.Parameter {
+	return ws.QueryParameter("force", "Flag for the forced actions like Restart and Delete").Required(false)
+}
+
+func GracePeriod(ws *restful.WebService) *restful.Parameter {
+	return ws.QueryParameter("graceperiod", "Param for the gracePeriod on opeartions like Restart and Delete").Required(false)
```

Comment:
typo "operations"

Suggest tweaking this too: "Grace period before action is forced. Applies to Restart and Delete"

Reply:
I'm not sure what our own convention is here. Should it be "gracePeriod"? @rmohr?

Reply:
Corrected the doc string here